ts/src/bitmart.ts

```diff
@@ -1874,7 +1876,62 @@ export default class bitmart extends Exchange {
         return this.parseTradingFee (data);
     }
 
+    parseOrderV4 (order, market = undefined) {
+        const id = this.safeString (order, 'orderId');
+        const clientOrderId = this.safeString (order, 'clientOrderId');
+        const timestamp = this.safeInteger (order, 'createTime');
+        const marketId = this.safeString (order, 'symbol');
+        const symbol = this.safeSymbol (marketId, market, '_');
+        let type = this.safeString (order, 'type');
+        const price = this.safeString (order, 'price');
+        const side = this.safeString (order, 'side');
+        const amount = this.safeString (order, 'size');
+        const filled = this.safeString (order, 'filledSize');
+        const average = this.safeString (order, 'priceAvg');
+        const state = this.safeString (order, 'state');
+        const status = this.parseOrderStatusByType ('spot', state, true);
+        let timeInForce = undefined;
+        let postOnly = undefined;
+        if (type === 'limit_maker') {
+            type = 'limit';
+            postOnly = true;
+            timeInForce = 'PO';
+        }
+        if (type === 'ioc') {
+            type = 'limit';
+            timeInForce = 'IOC';
+        }
+        return this.safeOrder ({
+            'id': id,
+            'clientOrderId': clientOrderId,
+            'info': order,
+            'timestamp': timestamp,
+            'datetime': this.iso8601 (timestamp),
+            'lastTradeTimestamp': undefined,
+            'symbol': symbol,
+            'type': type,
+            'timeInForce': timeInForce,
+            'postOnly': postOnly,
+            'side': side,
+            'price': price,
+            'stopPrice': undefined,
+            'triggerPrice': undefined,
```

Comment:
I guess it corresponds to the limit price, you could have this information, see this structure of an open order fetched with 
privatePostSpotV4QueryOpenOrders
```
{
  code: '1000',
  message: 'success',
  data: [
    {
      orderId: '172313417908931942',
      clientOrderId: '172313417908931942',
      symbol: 'CHEEL_USDT',
      side: 'buy',
      orderMode: 'spot',
      type: 'limit',
      state: 'new',
      price: '1.00000',
      priceAvg: '0.00000',
      size: '5.04834',
      filledSize: '0.00000',
      notional: '5.04834000',
      filledNotional: '0.00000000',
      createTime: '1692884134762',
      updateTime: '1692884134786'
    }
  ],
  trace: 'fbaee9e0e2f5442fba5b3262fc86b0ac.76.16928841599618443'
}
```